doc/source/scalability/innodb_io.rst

```diff
@@ -65,6 +65,10 @@ The following values are allowed:
   * ``ALL_O_DIRECT``: 
     use O_DIRECT to open both data and log files, and use ``fsync()`` to flush the data files but not the log files. This option is recommended when |InnoDB| log files are big (more than 8GB), otherwise there might be even a performance degradation. **Note**: When using this option on *ext4* filesystem variable `innodb_log_write_ahead_size <https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_log_write_ahead_size>`_ should be set to 4096 (default log-block-size in *ext4*) in order to avoid the ``unaligned AIO/DIO`` warnings.
 
+Interaction with :ref:`parallel_doublewrite_buffer`
+---------------------------------------------------
+
+Regardless of :variable:`innodb_flush_method setting`, the parallel doublewrite file is opened with ``O_DIRECT`` flag to remove OS caching, then its access is further governed by the exact value set: if it's set to ``O_DSYNC``, the parallel doublewrite is opened with ``O_SYNC`` flag too. Further, if it's one of ``O_DSYNC``, ``O_DIRECT_NO_FSYNC``, or ``ALL_O_DIRECT``, then the doublewrite file is not flushed after a batch of writes to it is completed.
```

Comment:
O_DIRECT bit may be removed and the rest inlined into the value descriptions above